docs/ur/README.md

```diff
@@ -0,0 +1,351 @@
+
+| Name | Format | Example |
+| ---- | ------ | ------- |
+| Short | mm/dd/yy | 12/31/19 |
+| Abbreviated | month dd | مارچ 31 |
+
+### Days
+
+| Day | Urdu | Abbreviation |
+| ----  | ------ | ------- |
+|Monday | پیر |           n/a|
+|Tuesday  |    منگل     | n/a|
+|Wednesday|     بدھ     | n/a|
+|Thursday |   جمعرات    | n/a|
+|Friday   |    جمعہ     | n/a|
+|Saturday | ہفتہ        | n/a|
+|Sunday   | اتوار       | n/a|
+
+### Time format
+
+Time should be written as Standard format.
+
+```
+
+i.e. hh:mm:ss AM/PM or HH:MM
+03:24:12 PM
+14:32
+
+```
+
+### Numerals
+
+Numbers should be same as english numbers.
+
+```
+
+* Example: 2 will be 2 and not ٢
+* Wrong: ١٢٣٤٥٦٧٨٩١٠
+* Correct : 12345678910
+
+```
+
+ | Separator | Character Name | Symbol | Example |
+ | --------- | -------------- | ------ | ------- |
+ | Decimal | Period | `.` | 1.23 |
+ | Thousands | Comma | `,` | 1,234 |
+ | Percentage | Percent sign | `%` | 99.95% |
+
+### Currency
+
+> Currency in Urdu is Rupee - روپیہ
+
+```
+
+* Rupee - روپیہ
+* روپیہ 100
+
+```
+
+### Address and postal code format
+
+```
+
+[addressee]
+[street number and name][building and suite numbers]
+[district][city][state/province][postal code]
+[Country]
+
+```
+
+### Telephone number format
+
+```
+
+* +1/+91/+92/ XXXXXXXXXX
+* e.g +1 1234567890
+
+```
+
+### Units of measurement
+
+| English | Translation | Abbreviation |
+| ------- | ----------- | ------------ |
+| Kilometer |      کلو میٹر         |Not Applicable|
+| Meter|         میٹر              |Not Applicable|
+|Decimeter|    ڈیسی  میٹر          |Not Applicable|
+|Centimeter|     سینٹی میٹر        |Not Applicable|
+|Millimeter|     ملی میٹر          |Not Applicable|
+|Hectoliter|     ہیکٹو لیٹر        |Not Applicable|
+|Liter    |     لیٹر               |Not Applicable|
+|Deciliter|      ڈیسی لیٹر         |Not Applicable|
+|Centiliter|      سینٹی لیٹر       |Not Applicable|
+|Milliliter|      ملی لیٹر         |Not Applicable|
+|Ton    |     ٹن                    |Not Applicable|
+|Kilogram|       کلو گرام          |Not Applicable|
+|Pound    |     پاؤنڈ              |Not Applicable|
+|Gram    |     گرام                |Not Applicable|
+|Decigram|      ڈیسی گرام          |Not Applicable|
+|Centigram|     سینٹی گرام         |Not Applicable|
+|Milligram|     ملی گرام           |Not Applicable|
+|Inch    |     انچ                 |Not Applicable|
+|Feet    |     فٹ                  |Not Applicable|
+|Mile     |    میل                 |Not Applicable|
+|Gallon |    گیلن                  |Not Applicable|
+
+## Grammar, Syntax & Orthographic Conventions
+
+> This section includes information on how to apply the general rules of the Urdu language to Mozilla Products.
+
+### Adjective
+
+In Urdu, adjectives should be handled in the following manner.
+
+Adjectives are also considered as nouns.
+
+**Nouns should not be translated but transliterated, except for Brand Names.**
```

Comment:
Can you explain this to me please? Why would nouns not be translated?

Reply:
A noun is a name of person, place or a thing. And in Urdu, we don't translate the name, we transliterate it.
Example:
Name: Shahbaz
Translation: پرندوں کا بادشاہ
Transliterate: شہباز

Reply:
I understand why that would apply to people, but "window" is also a noun. Why would that not be translated?

Reply:
Should we go ahead with this?

Reply:
If we translate window in Urdu, it would be کھڑکی which is the window in homes, so to eliminate that with pop-up window, we decided to keep it as ونڈو

Reply:
I'm sorry, I don't know Urdu so I don't know the difference between the two terms you put there. In English, the term "window" in this context is meant to be symbolic of the window of a house, just FYI. 

I suppose I still don't understand why a noun like, "window" or "toolbar" or "address bar" or "preferences" would be transliterated, not translated. Presumably, the user would have no idea what "preferences" or other nouns means, but would be able to pronounce them vocally.

Reply:
Sorry for giving reference in Urdu. 

During the meetup, we decided to go for transliteration of nouns, as this is most familiar in both the country and translating noun such as preferences, address bar, toolbars make the localisation more complex and lose it's meaning.

Hence, we decided to go for transliteration. 

Let me ask team, what they think of this.

Reply:
I'm happy to be involved in this discussion, if it helps. It's our position that these things should be translated, as translation is a meaning transfer, not a phonetic or typographic transfer. For terms that don't have equivalents in Urdu, there are many communities to use neologisms to coin new terms in the target language. I'd be happy to connect you with any of them to learn how they've done this in a way that users will understand.

Reply:
Yes, definitely.

Let's schedule a call or a discourse post or issue here on GitHub, to make this point clear.

Reply:
Whatever you feel is most appropriate. Just be sure to invite me :-)

Reply:
Sure.

Since few of the members are still travelling way back, will schedule a call after that. 
We are also starting Urdu Team Monthly call on every third Saturday @ 9:30 PM IST / 9:00 PM PKT

Reply:
If you feel that the monthly call would be a good place for this discussion, I'm happy to join. I just don't want to take away everyone being able to hold the meeting in Urdu.

Reply:
Thanks for the suggestion. As this is going to be the first call after making of the stylesheet, so everyone agreed to have a separate call with you.

Sent you an invite to the call.

**Date&Time**: Jan 16 2020, Thursday 9:00 PM IST

Please confirm.

Reply:
Hi @gueroJeff 

Update PR, please review.

Noun section will come in the next update.